src/solver/surfaceIntegrations.F90

```diff
@@ -449,6 +503,32 @@ subroutine wallIntegrationFace(localValues, mm)
        Mp(2) = Mp(2) + my*blk
        Mp(3) = Mp(3) + mz*blk
 
+       ! the force integral for the center of pressure computation.
+       ! We need the cell centers wrt origin
+       xc = fourth*(xx(i,j,  1) + xx(i+1,j,  1) &
+            +         xx(i,j+1,1) + xx(i+1,j+1,1))
+       yc = fourth*(xx(i,j,  2) + xx(i+1,j,  2) &
+            +         xx(i,j+1,2) + xx(i+1,j+1,2))
+       zc = fourth*(xx(i,j,  3) + xx(i+1,j,  3) &
+            +         xx(i,j+1,3) + xx(i+1,j+1,3))
+
+       ! accumulate in the sums. each force component is tracked separately
+
+       ! Force-X
+       COFSumFx(1) = COFSumFx(1) + xc * fx * blk
+       COFSumFx(2) = COFSumFx(2) + yc * fx * blk
+       COFSumFx(3) = COFSumFx(3) + zc * fx * blk
```

Comment:
Do similar calculations also have to be added to zipperIntegrations.F90? The center of force test is nice, but it does not test the zipper integrations.